source/code-of-conduct.rst

```diff
@@ -1,44 +1,13 @@
 .. _`Code of Conduct`:
 
-===========================
-Contributor Code of Conduct
-===========================
+===================
+PSF Code of Conduct
+===================
```

Comment:
Since this document isn't the "PSF Code of Conduct", and merely links to it...

```suggestion
===============
Code of Conduct
===============
```

Reply:
Furthermore, I think this should either be identical to the file from https://github.com/pypa/.github/pull/1/ or just link directly to it (so we don't have to keep it updated it in two places).